environment.yml

```
@@ -21,6 +23,7 @@ dependencies:
   - numpy>=1.23
   - pillow>=8
   - pkg-config
+  - python
   - pygobject
   - pyparsing>=2.3.1
   - pyqt
```

place alphabetically?
```suggestion
  - pygobject
  - pyparsing>=2.3.1
  - pyqt
  - python
```